traitsui/testing/tester/_ui_tester_registry/qt4/_traitsui/instance_editor.py

```diff
@@ -41,6 +46,38 @@ def _get_nested_ui_custom(target):
     return target._ui
 
 
+def _get_combobox(target):
+    """ Obtains a nested combobox within an Instance Editor.
+
+    Parameters
+    ----------
+    target : instance of CustomEditor
+    """
+    return target._choice
```

Comment:
The only problem here is that there are situations where an instance editor will never set `_choice`.  This could lead to confusing bugs as someone could call `help()` on the wrapper objects in their test and see that they have the option to call `locate(Index(_)).perform(MouseClick())` only to find it failing.  e.g. in `test_instance_editor.TestInstanceEditor.test_custom_editor` I added:
```
value_ = tester.find_by_name(ui, "inst")
value_.help()
value_.locate(Index(1)).perform(MouseClick())
value_txt = value_.find_by_name("value")
```
and I see the following failure:

```
======================================================================
ERROR: test_custom_editor (test_instance_editor.TestInstanceEditor)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/aayres/Desktop/traitsui/traitsui/tests/editors/test_instance_editor.py", line 105, in test_custom_editor
    value_.locate(Index(1)).perform(MouseClick())
  File "/Users/aayres/Desktop/traitsui/traitsui/testing/tester/ui_wrapper.py", line 244, in perform
    self._perform_or_inspect(interaction)
  File "/Users/aayres/Desktop/traitsui/traitsui/testing/tester/ui_wrapper.py", line 310, in _perform_or_inspect
    return handler(self, interaction)
  File "/Users/aayres/Desktop/traitsui/traitsui/testing/tester/_ui_tester_registry/qt4/_traitsui/instance_editor.py", line 63, in _click_combobox_index
    delay=wrapper.delay,
  File "/Users/aayres/Desktop/traitsui/traitsui/testing/tester/_ui_tester_registry/qt4/_interaction_helpers.py", line 220, in mouse_click_combobox
    q_model_index = combobox.model().index(index, 0)
AttributeError: 'NoneType' object has no attribute 'model'

----------------------------------------------------------------------
``` 
but the call to `help()` suggests I can call `locate(Index)`.  Even further, calling `value_.locate(Index(1)).help()` says that that object can perform a `MouseClick` even though it ultimately wraps `None`.

I'm not sure the best way to work around this, and it is a more general problem to think about with the current registry framework: what do we do when an editor supports interactions _only some of the time_?